clang/lib/Parse/ParseTemplate.cpp

```diff
@@ -787,6 +787,20 @@ NamedDecl *Parser::ParseTemplateTemplateParameter(unsigned Depth,
                                                   unsigned Position) {
   assert(Tok.is(tok::kw_template) && "Expected 'template' keyword");
 
+  if (Token ahead = GetLookAheadToken(1); ahead.isNot(tok::less)) {
```

Comment:
One more thing, I somehow forgot to comment on this earlier, but we typically use PascalCase for variables, so this should be `Ahead` (though I’d probably call it `Next` or sth, but that’s not that important...).

Reply:
Actually this prompted me to check and I was just writing `NextToken()` manually (didnt realise `GetLookAheadToken` did a `-1`), so I think I'll switch it to `NextToken()` instead.
Then the renaming makes sense too with `Token Next = NextToken();`

Reply:
Ah yes, you can tell I don’t review parser code too often because I probably should have noticed that haha.